benchmark/single-source/ObjectiveCBridging.swift

```diff
@@ -744,3 +747,29 @@ public func run_ASCIIStringFromFile(_ N: Int) {
   #endif
 }
 
+var UnicodeStringFromCodable:String? = nil
+var UnicodeStringFromCodableDict = [String:Void]()
+public func setup_UnicodeStringFromCodable() {
+  do {
+    let jsonString = "[\(String(reflecting: "Nice string which works rather slöw."))]"
+
+    let decoded = try JSONDecoder().decode([String].self, from: Data(jsonString.utf8))
+    let reEncoded = try JSONEncoder().encode(decoded)
+    let desc = try JSONDecoder().decode([String].self, from: reEncoded)
+
+    UnicodeStringFromCodable = desc[0]
+  } catch (_) {
+    
+  }
+}
+
+@inline(never)
+public func run_UnicodeStringFromCodable(_ N: Int) {
+  #if _runtime(_ObjC)
+  for _ in 0 ..< N {
+    for _ in 0..<100 {
+      UnicodeStringFromCodableDict[UnicodeStringFromCodable!] = ()
```

Comment:
It may be worth passing the key through `identity()` to defeat a Sufficiently Advanced Optimizer.